src/components/DatePicker/README.md

```diff
@@ -86,3 +86,39 @@ function DatePickerExample() {
 ![Date picker on iOS](/public_images/components/DatePicker/ios/default@2x.png)
 
 <!-- /content-for -->
+
+---
+
+## Accessibility
+
+<!-- content-for: android -->
+
+See Material Design and development documentation about accessibility for Android:
+
+- [Accessible design on Android](https://material.io/design/usability/accessibility.html)
+- [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility/)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
+
+- [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/accessibility/)
+- [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+<!-- /content-for -->
+
+<!-- content-for: web -->
+
+Some users might find interacting with date pickers to be challenging. When you use the [date picker component](/components/forms/text-field), always give users the option to enter the date using a text field component as well.
```

Comment:
The only thing I would tweak is taking out this link, which I think it probably a stray copy/paste:

```suggestion
Some users might find interacting with date pickers to be challenging. When you use the date picker component, always give users the option to enter the date using a text field component as well.
```

Otherwise, this is great!

Reply:
Ohhh lol! Nice catch